tests/integration/targets/monit/files/httpd_echo.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
```

Comment:
Same here. Where does this file comes from? It should have copyright and license header.

Reply:
I wrote this, I'll put a GPL v3 on it.